### ISSUE TITLE
Readme

### ISSUE THREAD

**Issue by ssweber** [PySimpleSQL/pysimplesql#198]
Are you ok if I chip away at reorganizing the readme and adding screenshots/short animated gifs for the features? I plan on adding a linked-table of contents. I will keep in my branch and then if you like, move over.

Short getting started: Sql (with fk, update/delete cascade) + layout = gui

Features
- Design user interfaces with automatic relationship handling - parent/child linking, cascade description
- Dynamic comboboxes - add
- Sortable tables - gif of sorting action
- Customizable button themes - show full list, possibly make an alternative look to include. Themepacks
- Localization - LanguagePacks
- Edit protection - toggle edit protect gif
- Prompt Save / autosave modes. gif showing popup on record change

- Fields: 
- Pre-populate new record windows with database-defined defaults - gif w/ side-by-side sql gist
- Display 'required' markers - gif w/ side-by-side sql gist
- Automatically fill in not-null fields based on column data types. gif w/ side by side sql gist

Advanced topics:
Data transforms - code, maybe show database values side by side
Callbacks - edit protect example
Event mapping ??

Working with Databases
Sqldriver
ResultSet

Explanations of behind-the-scenes
All your great explanations of how it pysimplesql automatically does things behind the scenes.

**Comment by PySimpleSQL** [comment]
Absolutely, if you have time and want to dig into it that would be great

**Comment by ssweber** [comment]
I haven’t forgotten about this, just got a little side-tracked with black/ruff. I plan on doing some this weekend.

**Comment by ssweber** [comment]
putting this here for brain-storming how to present a short landing-page readme.

### pysimplesql top-5 features:

1. Binds PySimpleGUI widgets to your data, via specialized SqlDrivers
2. Creates an internal model of database tables. No need to redefine, just point at an existing db or provide sql to create.
3. Pre-made buttons for common Form actions - Lock / Save / (First/Previous/Next/Last) / Duplicate / New / Delete
4. Automatic relationship handling: `select` a record, and linked/dependent records are re-queried and their widgets updated.
5. Populates Combo Dropdowns and Table widgets with information extracted from Relationship data. Eg. Store a `parent_id` key as `1`, but display as their name.

### New features in Development branch:
- Data managed as Pandas Dataframes
- Enhanced Table widget: Sort columns, edit cells, filter via search, lazily load rows.
- Enhanced widgets. Add placeholders to Input/Multiline. Autocomplete combobox.
- Live update. Edit a cell and individual widgets will update (and vise-versa).
- Validate while typing, on save, or both! Specialized ColumnClasses are extendable with additional validation constraints.
